docs/updates/2019-05.md

```diff
@@ -0,0 +1,104 @@
+# May 2019
+
+Welcome to the May 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Release of corectl 1.0.0
+
+After a final touch up we released corectl as a stable 1.0.0. As described in
+[previous update](2019-04.md#corectl-100-beta-release) we changed the command structure and with this release we
+confirmed it.
+
+### Installation through snap
+
+For our Linux users out there we have added support for installation of latest corectl through
+[snap](https://snapcraft.io/). Install with:
+
+```bash
+snap install qlik-corectl
+```
+
+and update
+
+```bash
+snap refresh qlik-corectl
+```
+
+Even though the package name has a prefix `qlik-` the installed binary will be called corectl.
+
+### Support for multiple entities in files
+
+We have added support for having multiple entities in object files (measures, dimensions and objects). Depending on
+your preference you can add multiple objects in one json file or divide them into several.
+
+```json
+[
+    {
+        "qInfo": {
+            "qId": "avgSize",
+            // ...
+        }
+    },
+    {
+        "qInfo": {
+            "qId": "entries",
+            // ...
+        }
+    }
+]
+```
+
+### Other fixes
+
+* Fix bash completion in Widows Subsystem for Linux (WSL)
+* Additional information in the version command
+
+## New file streaming API in engine
+
+A new file streaming API has been released in engine, giving us a new way of working with remote files.
```

Comment:
I think we should be consistent with the naming of this API. In the documentation we refer to it by `File Connector API`. Maybe also attach a link to the name pointing to the API introduction i.e. https://core.qlik.com/services/qix-engine/apis/file-loading/introduction/